app/models/submission.rb

```diff
@@ -134,6 +139,14 @@ def polish_language?
     self.language == POLISH
   end
 
+  def version_author version
+    if !version.whodunnit.nil?
```

Comment:
Widzę, że to mechanizm paper_trail. Można to zostawić, choć nie jest to zbyt poprawne.